joomla/joomla-cms
# mod_menu: Allowing to exclude menu items from displaying in frontend menus

#### Description

This PR adds the ability to exclude menu items from displaying in frontend in mod_menu.

Simple use case:
We have a menu where we have 5 items: Home; The company; Services; Portfolio; Contacts
But we don't want to show "Home" in the main menu, just: The company; Services; Portfolio; Contacts.

This PR will also allow to create "hidden" menu items that exist in the menu and the website but aren't displayed in the main menu without having to create another menu for those "hidden" menu items.
#### How to test
1. Fresh Joomla install with multilanguage.
2. Apply this patch
3. Go to the main menu module and use the new field "Exclude Menu Items" to exclude some menu items from displaying in the frontend menu.
4. Check in the frontend it those items are not displayed in the main menu.
#### Observations

Suggestions are welcome.
